### PR TITLE
fixed the segmentation relabel bug and pre-trained model loader bug

### PR DESCRIPTION
**Pull request by divyam-goel**

### CODE REVIEW

File: connectomics/data/utils/data_segmentation.py [hunk 1]
```diff
@@ -29,7 +29,7 @@ def relabel(seg, do_type=False):
     if do_type:
         m_type = getSegType(mid)
     mapping = np.zeros(mid, dtype=m_type)
-    mapping[uid] = np.arange(len(uid), dtype=m_type)
+    mapping[uid] = np.arange(1, len(uid) + 1, dtype=m_type)
```
Comment (donglaiw) [inline connectomics/data/utils/data_segmentation.py:32]: error case: uid has 0

Comment (donglaiw) [reply connectomics/data/utils/data_segmentation.py:32]: uid=uid[uid>0]

Comment (divyam-goel) [reply connectomics/data/utils/data_segmentation.py:32]: I tried reverting this change in line 32, and instead added `uid=uid[uid>0]`, but the problem persists. So should I add `uid=uid[uid>0]` without reverting the line 32 change?

Comment (donglaiw) [reply connectomics/data/utils/data_segmentation.py:32]: the solution should be: add uid=uid[uid>0] and then your change (numbering things from 1)

**Comment by zudi-lin** [comment]
hi @divyam-goel, could you please reverse the commit related to **--checkpoint** (commit c8c9d19)? I am designing a new logic to handle checkpoint I/O and the original model loading functions in `connectomics/model/__init__.py` will be deprecated. I will merge the fix of the relabeling bugs after your revision.

**Comment by divyam-goel** [comment]
hello @zudi-lin, I have reverted the checkpoint change. Please merge the relabel bug fix.

**Pull request merged**